examples/compile_efficientnet.py

```diff
@@ -5,39 +5,29 @@
 import ast
 
 def compile_net(run, special_names, statement_builder = lambda name, cargs, global_size: f"{name}({', '.join(cargs)});"):
-  # functions that run the net
-  functions = {}
-  bufs = {}
-  bufnum = 0
-  statements = []
-  bufs_to_save = {}
+  functions, bufs, bufs_to_save, statements, bufnum = {}, {}, {}, [], 0
   for fxn,args in run.jit_cache:
     functions[fxn.name] = fxn.prg   # NOTE: this assumes all with the same name are the same
     cargs = []
     for i,arg in enumerate(args):
       key = id(arg)
       if key not in bufs:
         if key in special_names:
-          bufs[key] = (special_names[key], len(arg._buf))
+          bufs[key] = (special_names[key], arg._buf.size, key)
         else:
-          bufs[key] = (f"buf_{bufnum}", len(arg._buf))
+          bufs[key] = (f"buf_{bufnum}", arg._buf.size, key)
           bufnum += 1
           if i > 0: bufs_to_save[bufs[key][0]] = arg   # if first usage of a buffer is not an output, and it's not a special name
```

Comment:
this breaks the tests in ci as arg._buf doesn't have the method size

Reply:
I'll fix it